templates/chisel/nix/pkgs/vcs-fhs-env.nix

```diff
@@ -59,10 +59,10 @@ buildFHSEnv {
     sqlite
     nssmdns
     (libkrb5.overrideAttrs rec {
-      version = "1.18.2";
+      version = "1.21.3";
       src = fetchurl {
         url = "https://kerberos.org/dist/krb5/${lib.versions.majorMinor version}/krb5-${version}.tar.gz";
-        hash = "sha256-xuTJ7BqYFBw/XWbd8aE1VJBQyfq06aRiDumyIIWHOuA=";
+        hash = "sha256-t6TNXq1n+wi5gLIavRUP9yF+heoyDJ7QxtrdMEhArTU=";
```

Comment:
Not required, this version pining is for VCS runtime.

Reply:
This version will break vcs-fhs-env
See https://github.com/chipsalliance/chisel-nix/actions/runs/10856473151/job/30131167984

Reply:
The better option is provide a nixpkgs version to `vcs-fhs-env` and pin it. See what have been done for circt.

Reply:
Try this: https://github.com/chipsalliance/t1/blob/ab3279151c3c33190056a4af890541fef4d79b33/nix/pkgs/vcs-fhs-env.nix#L58-L70